test/rekt/features/channel/features.go

```diff
@@ -93,3 +95,54 @@ func DeadLetterSink() *feature.Feature {
 
 	return f
 }
+
+func EventTransformation() *feature.Feature {
+	f := feature.NewFeature()
+	lib := feature.MakeRandomK8sName("lib")
+	channel1 := feature.MakeRandomK8sName("channel 1")
+	channel2 := feature.MakeRandomK8sName("channel 2")
+	prober := eventshub.NewProber()
+	prober.SetTargetResource(channel_impl.GVR(), channel1)
+
+	f.Setup("install events", eventlibrary.Install(lib))
+	f.Setup("use events cache", prober.SenderEventsFromSVC(lib, "events/three.ce"))
```

Comment:
this step also does not have to be in the setup phase, it is just configuring the probers state